src/main/kotlin/com/dansplugins/fiefs/command/fiefs/FiefsCommand.kt

```diff
@@ -42,8 +49,10 @@ class FiefsCommand(private val plugin: Fiefs) : CommandExecutor, TabCompleter {
             1 -> subcommands.filter { it.startsWith(args[0]) }.toMutableList()
             else -> when(args.first().lowercase()) {
                 "help" -> helpCommand.onTabComplete(sender, command, alias, args.drop(1).toTypedArray())
+                "create" -> createCommand.onTabComplete(sender, command, alias, args.drop(1).toTypedArray())
+                "list" -> listCommand.onTabComplete(sender, command, alias, args.drop(1).toTypedArray())
                 else -> emptyList()
-            }
+            } as List<String>
```

Comment:
Cast shouldn't be necessary here, all of the onTabComplete methods should return a list.

Reply:
See https://github.com/Dans-Plugins/Fiefs/pull/135#discussion_r1133190028

Reply:
This has been removed!